src/Control/Controller.php

```diff
@@ -397,8 +397,6 @@ public function getViewer($action)
             $templates = $this->templates[$action];
         } elseif (isset($this->templates['index']) && $this->templates['index']) {
             $templates = $this->templates['index'];
-        } elseif ($this->template) {
```

Comment:
This variable isn't defined. In theory people could define it in a subclass, but it's far from being a published API. This code goes back to the project's inception and I think it's an oversight.

In Security.php, private static $template is defined, which does *not* populate this variable - it's used as a config var. Notably, this code on PHP 7.4 failed to print HELLO

```
<?php

class A {
  function foo() {
    return $this->template;
  }
}

class B extends A {
  static $template = 'HELLO';
}

echo (new B())->foo();
```

In my view, removing this isn't a breaking change, but it's moderately controversial. As an alternative I would add a /** @phpstan-ignore-next-line */ flag; attempting to define template breaks other things.